### PR TITLE
Dev1.2.2

### PR DESCRIPTION
**Pull request by climbage**
Cross-reference #109

### CODE REVIEW

**Comment by randallwhitman** [comment]
Is release 1.2.2 or 1.3 waiting on anything specific?

**Comment by stolstov** [comment]
@climbage This is ready to be tagged and released.

**Comment by randallwhitman** [comment]
Excellent!

**Comment by climbage** [comment]
Sounds good.  I'll tag and push to maven next week.  Do we have a rough change list for the release?

**Comment by stolstov** [comment]
#112 #110 #106 #104 #101 #85.
The GeoJSON export was rewritten. There was also a bug in the Buffer code that made buffer a little larger than needed on concave corners.

File: src/main/java/com/esri/core/geometry/ogc/OGCGeometry.java [hunk 1]
```diff
@@ -489,7 +496,7 @@ public static OGCGeometry fromEsriShape(ByteBuffer buffer) {
 	}
 
 	public static OGCGeometry fromJson(String string)
-			throws JsonParseException, IOException {
+			throws Exception {
```
Comment (randallwhitman) [inline src/main/java/com/esri/core/geometry/ogc/OGCGeometry.java]: Just now noticed, this is an interface change which will require changes to callers.

**Pull request merged**